modules/ROOT/pages/clustering/servers.adoc

```diff
@@ -88,8 +88,10 @@ Additionally, servers which are deallocating will not have any further databases
 [role=label--new-5.15]
 === Deallocated state
 
-When a server is in the deallocated state it no longer hosts any databases any databases besides system and can be removed from the cluster.
+When a server is in the deallocated state it no longer hosts any databases besides system and can be removed from the cluster.
 Additionally, deallocated servers cannot have any further databases allocated to them.
+Note, there is a known situation in which a previously deallocated offline server can transiently show as deallocating when restarting, it will however eventually return to the 
```

Comment:
```suggestion
Note that there is a known situation in which a previously deallocated offline server can transiently show as deallocating when restarting, it will, however, eventually return to the deallocated state without intervention.
```

Reply:
no sentence splitting in the docs